templates/catalog/product.tpl

```diff
@@ -301,32 +301,11 @@
             {/if}
           {/block}
 
-          {block name='product_customization'}
-            {if $product.is_customizable}
-              <section class="product-customization">
-                <h3>{l s='Product customization'}</h3>
-                <form method="post" action="{$customizationFormTarget}" enctype="multipart/form-data">
-                  <ul>
-                    {foreach from=$product.customizations.fields item="field"}
-                      <li>
-                        <label>{$field.label}</label>
-                        {if $field.type == 'text'}
-                          <textarea {if $field.required} required {/if} name="{$field.input_name}">{$field.text}</textarea>
-                        {elseif $field.type == 'image'}
-                          {if $field.is_customized}
-                            <img src="{$field.image.small.url}">
-                            <a class="remove-image" href="{$field.remove_image_url}" rel="nofollow">{l s='Remove Image'}</a>
-                          {/if}
-                          <input {if $field.required} required {/if} type="file" name="{$field.input_name}">
-                        {/if}
-                      </li>
-                    {/foreach}
-                  </ul>
-                  <button name="submitCustomizedDatas">{l s='Save Customization'}</button>
-                </form>
-              </section>
-            {/if}
-          {/block}
+          {if $product.is_customizable}
```

Comment:
don't forget to refactor this part.
Also, what happens if we want to display a message when the product is not customizable ?